### PR TITLE
convert to v1beta1 clusteraddons

### PR DESCRIPTION
**Pull request by s12chung**

### CODE REVIEW

File: templates/dex-k8s-authenticator.yaml [hunk 1]
```diff
@@ -20,8 +20,8 @@ spec:
     - name: none
       enabled: true
   requires:
-    matchLabels:
-      kubeaddons.mesosphere.io/name: dex
+    - matchLabels:
```
Comment (jieyu) [inline templates/dex-k8s-authenticator.yaml]: This also depends on traefik (ingresscontroller)

File: templates/kube-oidc-proxy.yaml [hunk 2]
```diff
@@ -20,8 +20,8 @@ spec:
     - name: none
       enabled: true
   requires:
-    matchLabels:
-      kubeaddons.mesosphere.io/provides: ingresscontroller
+    - matchLabels:
```
Comment (jieyu) [inline templates/kube-oidc-proxy.yaml]: This also depends on dex and cert-manager

File: templates/traefik-forward-auth.yaml [hunk 3]
```diff
@@ -9,8 +9,8 @@ spec:
   kubernetes:
     minSupportedVersion: v1.15.0
   requires:
-    matchLabels:
-      kubeaddons.mesosphere.io/name: dex
+    - matchLabels:
```
Comment (jieyu) [inline templates/traefik-forward-auth.yaml]: This also depends on ingresscontroller

File: templates/dex-k8s-authenticator.yaml (hunk 1 above)
Comment (s12chung) [reply templates/dex-k8s-authenticator.yaml]: done.

File: templates/kube-oidc-proxy.yaml (hunk 2 above)
Comment (s12chung) [reply templates/kube-oidc-proxy.yaml]: done.

File: templates/traefik-forward-auth.yaml (hunk 3 above)
Comment (s12chung) [reply templates/traefik-forward-auth.yaml]: done.

**Pull request merged**